plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -48,6 +57,9 @@ export function useProductHelper() {
 		draft: false,
 		publish: false,
 	} );
+	const context = useContext( CurrencyContext );
+	const { getCurrencyConfig } = context;
+	const { decimalSeparator } = getCurrencyConfig();
```

Comment:
Should we move the above two lines into the `sanitizePrice` function? given the `decimealSeperator` isn't used anywhere else, and would needlessly be triggered on every re-render. It's probably not super JS extensive, but it be nice to keep that stuff to a minimum.

Reply:
Good idea! I changed it in commit [b60ef79](https://github.com/woocommerce/woocommerce/pull/34558/commits/b60ef7953cb9944bed74553e8196cbe98bffb498)